Exiled.Events/Patches/Events/Player/ChangingIntercomMuteStatus.cs

```diff
@@ -22,25 +23,33 @@ internal static class ChangingIntercomMuteStatus
     {
         private static bool Prefix(CharacterClassManager __instance, bool value)
         {
-            ChangingIntercomMuteStatusEventArgs ev = new ChangingIntercomMuteStatusEventArgs(API.Features.Player.Get(__instance._hub), value, true);
+            try
+            {
+                ChangingIntercomMuteStatusEventArgs ev = new ChangingIntercomMuteStatusEventArgs(API.Features.Player.Get(__instance._hub), value, true);
 
-            Player.OnChangingIntercomMuteStatus(ev);
+                Player.OnChangingIntercomMuteStatus(ev);
 
-            if (!ev.IsAllowed)
-            {
-                if (value == true)
-                {
-                    MuteHandler.RevokePersistentMute("ICOM-" + __instance.UserId);
-                }
-                else
+                if (!ev.IsAllowed)
                 {
-                    MuteHandler.IssuePersistentMute("ICOM-" + __instance.UserId);
+                    if (value == true)
+                    {
+                        MuteHandler.RevokePersistentMute("ICOM-" + __instance.UserId);
+                    }
+                    else
+                    {
+                        MuteHandler.IssuePersistentMute("ICOM-" + __instance.UserId);
+                    }
+
+                    return false;
                 }
 
-                return false;
+                return true;
+            }
+            catch (Exception e)
+            {
+                Exiled.API.Features.Log.Error($"Exiled.Events.Patches.Events.Player.ChangingIntercomMuteStatus: {e}\n{e.StackTrace}");
```

Comment:
```suggestion
                Exiled.API.Features.Log.Error($"{typeof(ChangingMuteStatus).FullName}.{nameof(Prefix)}:\n{e}");
```
And do the same with another patch

Reply:
gotcha, copied this from a different patch